fe/src/setupTests.ts

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import '@testing-library/jest-dom'; // eslint-disable-line
```

Comment:
Eslint doesn't approve of `jest-dom` being devDependency, but it makes sense to keep it as it (?)
At least for now IMO, this problem can be rechecked when we get more into testing.